google-cloud-bigtable-deps-bom/pom.xml

```diff
@@ -73,7 +73,13 @@
       <dependency>
         <groupId>com.google.cloud</groupId>
         <artifactId>google-cloud-monitoring-bom</artifactId>
-        <version>3.6.0</version>
+        <version>3.7.0</version>
+        <exclusions>
```

Comment:
can you add a comment that we are using the perfmark version from opencensus (in all files)?

Reply:
Updated!